Documentation/observability/visibility.rst

```diff
@@ -12,63 +12,73 @@ Layer 7 Protocol Visibility
+    Historically, it had been possible to enable L7 visibility using Pod
+    annotations (``policy.cilium.io/proxy-visibility``). This method is
+    no longer supported and we recommend users to switch to L7 policies instead.
+
+The following example demonstrates how to enable DNS and HTTP visibility for all
+traffic within the ``default`` namespace. We do that by creating a ``CilimNetworkPolicy`` 
+resource containing two L7 rules -- one for DNS and one for HTTP traffic. 
+If all you're interested in is visibility, you can omit or wildcard
+the L7 matching conditions as it is shown below:
+
+.. code-block:: yaml
+
+      apiVersion: "cilium.io/v2"
+      kind: CiliumNetworkPolicy
+      metadata:
+        name: "l7-visibility"
+      spec:
+        endpointSelector:
+          matchLabels:
+            "k8s:io.kubernetes.pod.namespace": default
+        egress:
+        - toPorts:
+          - ports:
+            - port: "53"
+              protocol: ANY
+            rules:
+              dns:
+              - matchPattern: "*"
+        - toEndpoints:
+          - matchLabels:
+              "k8s:io.kubernetes.pod.namespace": default
+          toPorts:
+          - ports:
+            - port: "80"
+              protocol: TCP
+            - port: "8080"
+              protocol: TCP
+            rules:
+              http: [{}]
+        - toEntities:
+          - all
+
+The final rule allowing all egress traffic (``toEntities: [ all ]``) is
+optional and can be removed or replaced with a more specific set of rules.
+
+Based on the above policy, Cilium will pick up all UDP/53, TCP/80 and TCP/8080 
+egress traffic from Pods in the ``default`` namespace and redirect it to the 
+proxy such that the output of ``cilium monitor`` or ``hubble observe`` shows 
+the L7 flow details:
 
 ::
 
-    -> Request http from 1474 ([k8s:id=app2 k8s:io.kubernetes.pod.namespace=default k8s:appSecond=true k8s:io.cilium.k8s.policy.cluster=default k8s:io.cilium.k8s.policy.serviceaccount=app2-account k8s:zgroup=testapp]) to 244 ([k8s:io.cilium.k8s.policy.cluster=default k8s:io.cilium.k8s.policy.serviceaccount=app1-account k8s:io.kubernetes.pod.namespace=default k8s:zgroup=testapp k8s:id=app1]), identity 30162->42462, verdict Forwarded GET http://app1-service/ => 0
-    -> Response http to 1474 ([k8s:zgroup=testapp k8s:id=app2 k8s:io.kubernetes.pod.namespace=default k8s:appSecond=true k8s:io.cilium.k8s.policy.cluster=default k8s:io.cilium.k8s.policy.serviceaccount=app2-account]) from 244 ([k8s:io.cilium.k8s.policy.serviceaccount=app1-account k8s:io.kubernetes.pod.namespace=default k8s:zgroup=testapp k8s:id=app1 k8s:io.cilium.k8s.policy.cluster=default]), identity 30162->42462, verdict Forwarded GET http://app1-service/ => 200
+    default/testapp-5b9cc645cb-4slbs:45240 (ID:26450) -> kube-system/coredns-787d4945fb-bdmdq:53 (ID:9313) dns-request proxy FORWARDED (DNS Query web.default.svc.cluster.local. A)
```

Comment:
Can you add the command that was used to generate this output? I think it makes sense to be specific here, since I'm not sure if this is from `hubble observe` or `cilium monitor`